services/datamanager/builtin/file_deletion.go

```diff
@@ -0,0 +1,135 @@
+package builtin
+
+import (
+	"context"
+	"errors"
+	"io/fs"
+	"math"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"go.viam.com/rdk/logging"
+
+	"github.com/ricochet2200/go-disk-usage/du"
+	"go.viam.com/rdk/services/datamanager/datacapture"
+	"go.viam.com/rdk/services/datamanager/datasync"
+)
+
+// TODO change these values back to what they should be
+const (
+	fsThresholdToTriggerDeletion = .8
+	captureDirToFSUsageRatio     = .5
+	n                            = 4
```

Comment:
Perhaps deleteEveryNth? Thanks for the naming updates, especially to make global consts readable.